config/settings/common.py

```diff
@@ -205,6 +222,19 @@
 # ------------------------------------------------------------------------------
 ROOT_URLCONF = "config.urls"
 
+# ADMIN_AREA_PREFIX = env("DJANGO_ADMIN_URL", default="admin/", cast=url_prefix)
+
+RESTRICTED_PREFIXES = env(
+    "RESTRICTED_PREFIXES", default=["admin/", "api/"], cast=url_prefix_list
```

Comment:
On line 273 we have:
```
ADMIN_URL = env("DJANGO_ADMIN_URL", default="admin")
ADMIN_URL_ROUTE = r"^{}/".format(ADMIN_URL)
```
and we do set this environment variable in production. ADMIN_URL_ROUTE is then used in urls.py to configure the admin route.

metadeploy does the same thing but called the setting ADMIN_AREA_PREFIX instead of ADMIN_URL_ROUTE. Which is why the middleware checks for the ADMIN_AREA_PREFIX setting. I think we should rename ADMIN_URL_ROUTE to ADMIN_AREA_PREFIX in MetaCI too, and then we can remove `admin/` from the defaults here.

Reply:
Can you clarify whether you want to rename only the SETTINGS variable or also the Environment variable? And if the latter, wouldn't that be a breaking change?

Reply:
Yeah, just the setting

Reply:
Maybe an alias is safer for backwards compatibility? 

It is a settings file, after all, someone could theoretically have their own clone of production.py.

Maybe its just a theoretical rather than a practical concern?

Reply:
That's a fair point. Adding an alias sounds reasonable.